app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -849,7 +849,7 @@ def product_data(product)
       plan_type: capitalize_value(product.plan_types).join(', '),
       pvp_areas: product.premium_value_products.map{ |pvp| pvp.rating_area.human_exchange_provided_code }.uniq.compact.join(',').presence || 'N/A',
       plan_id: product.hios_id,
-      metal_level_kind: product.metal_level_kind.to_s.capitalize
+      metal_level_kind: product.metal_level.to_s.capitalize
```

Comment:
Might want to update the key, as it's perhaps incorrectly specific now. But up to you.